Playbooks/playbook-DomainTools_Iris_Tags.yml

```diff
@@ -0,0 +1,145 @@
+id: DomainTools_Iris_Tags
```

Comment:
@idovandijk referencing my above comment to playbook.

The intent was to provide the user with a template (a micro-playbook as you suggest) that they can further extend. The use-case is if an Indicator/ observable is already known malicious, the playbook changes the severity of the Incident. However, user can customize this to send out a Service ticket or create human approval. The playbook demonstrates how user can parse through the tags and retrieve the metadata. (P.S. - We are extending the Tags metadata further in Q1, and this will have a lot more actionable details).

If this seems trivial enough to be considered a playbook, how would you suggest we package this? I would have considered command, but need to be automated as a playbook for scaleability.

Reply:
Hey there. The use-case is clear to me, but we cover that in already existing playbooks :)
In this case your script sets the severity according to the domain tags, but that could affect the severity of an existing incident for the worst, too. E.g., if you were to have a "critical" incident which gets a lower severity of "high" because of this script when it sees a bad tag in a domain.
The wa we handle this is we have the `Calculate Severity - Generic v2` playbook. Among other assessments, it looks at the DBotScore of all indicators of the incident and decides the final severity of the incident accordingly.
What you should do is output a DBotScore with this command instead, so if a domain with a bad tag was involved, then it will be handled and taken into account with all other IOCs in `Calculate Severity - Generic v2` automatically (not need to edit that generic playbook).

As for the metadata of tags - the answer is the same as I explained in one of my other comments. Your task provides the output data, and this data can then be used anywhere in any playbook and any task, so there is no need to tie a user specific to a flow where incident severity is affected immediately and solely based on domain tags.

If and when additional possible logic / flow / use case arises, regarding tags of domain, we will definitely be willing and happy to add it as an out-of-the-box playbook for customers.
If there is additional data that you would like to utilize from the CheckTags script, then it should output it as well as a DBotScore, but not set a severity.

To summarize: it seems that we agree that the tags should affect the severity. I'm saying that by outputting DBotScore, our existing OOTB playbooks will be able to take the bad tags into account when calculating a severity for the incident. As such, it seems that the purpose of this playbook is already covered elsewhere :)